### ISSUE TITLE
Page title is not updated as navigation takes place within dashboard and client-display

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1209]
The page/browser title should be updated as new content is loaded or navigated to. #1165 may be able to incorporate a solution to this.

**Comment by rmmayo** [comment]
PR: #1436

**Comment by mosefer** [comment]
created #1446 to account for global badges view not updating page title
